ethereum-explorer/src/util.rs

```diff
@@ -0,0 +1,23 @@
+use substreams::Hex;
+
+pub fn hexadecimal_to_string(hex: &Vec<u8>) -> String {
+    return Hex::encode(hex);
+}
+
+pub fn is_transaction_hash_valid(hash: &String) -> bool {
+    // A transaction hash is always 64 hexadecimal characters
+    if hash.len() != 64 {
+        return false;
+    }
+    
+    return true;
```

Comment:
We should support optionally `0x` in front.
```suggestion
    // A transaction hash is always 64 hexadecimal characters (or 66 if prepended with `0x`)
    hash.len() == 64 || hash.len() == 6
```